lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
Why is this failing though? IRB is included in the engines hash, but is obviously not being found. Whilst changing the way DEFAULT_ENGINE is defined / wrapped, is this still not going to fail? If IRB can't be found, it can't be found.